core/Db/Schema/Mysql.php

```diff
@@ -362,14 +362,14 @@ public function getTablesCreateSql()
             'changes'             => "CREATE TABLE `{$prefixTables}changes` (
                                       `idchange` INTEGER UNSIGNED NOT NULL AUTO_INCREMENT,
                                       `created_time` DATETIME NOT NULL,
-                                      `plugin_name` VARCHAR(255) NOT NULL,
+                                      `plugin_name` VARCHAR(40) NOT NULL,
                                       `version` VARCHAR(20) NOT NULL, 
                                       `title` VARCHAR(255) NOT NULL,                                      
                                       `description` TEXT NULL,
                                       `link_name` VARCHAR(255) NULL,
                                       `link` VARCHAR(255) NULL,       
                                       PRIMARY KEY(`idchange`),
-                                      UNIQUE KEY unique_plugin_version_title (`plugin_name`, `version`, `title`)                            
+                                      UNIQUE KEY unique_plugin_version_title (`plugin_name`, `version`, `title`(131))                            
```

Comment:
Should we make this like 100 then? For a title it's already very long.

Reply:
If we extend the plugin name to `60`, guess we can also lower the title to `100`

Reply:
I've updated the title index limit to 100